mllib/pom.xml

```diff
@@ -125,6 +125,10 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-test-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.janino</groupId>
+      <artifactId>janino</artifactId>
+    </dependency>
```

Comment:
I thought it would be better to explicitly depend on it now that we are using rather than depending on a transitive inclusion from another component.